src/utils/episodeSearchBoxFunc.ts

```diff
@@ -0,0 +1,17 @@
+import { IEpisode } from "../types";
+
+export function episodeSearchBoxFunc(
+  searchTerm: string,
+  data: IEpisode[]
+): IEpisode[] {
+  function callBackFunc(oneSetOfData: IEpisode): boolean {
+    if (oneSetOfData.summary !== null) {
+      return (
+        oneSetOfData.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
```

Comment:
Might be clearer to abstract these into variables to make it clearer what you are comparing.